spec/spec_helper.rb

```diff
@@ -21,4 +21,10 @@
 RSpec.configure do |config|
   config.infer_spec_type_from_file_location!
   config.use_transactional_fixtures = false
+
+  if defined?(ActiveStorage::Current)
+    config.before(:all) do
+      ActiveStorage::Current.host = 'https://www.example.com'
```

Comment:
Happened to me on another extension and it took a lot of time to understand what was happening. I opened https://github.com/rails/rails/pull/41996 to improve the dev experience here. 🤞